pandas/core/indexes/base.py

```diff
@@ -2529,15 +2529,20 @@ def set_value(self, arr, key, value):
     def _get_level_values(self, level):
         """
         Return an Index of values for requested level, equal to the length
-        of the index
+        of the index.
 
         Parameters
         ----------
-        level : int
+        level : int or level name
```

Comment:
The general format is to specify the data type on the first line followed by an explanation for their semantics in a subsequent block.

Reply:
Ok, I've changed this, also for MultiIndex.